blindml/frontend/config/task/task.py

```diff
@@ -101,38 +105,35 @@ def __init__(self, task_fp) -> None:
         )
 
     def search_for_model(self):
-        # this will resume? if experiment already exists?
-        run_nni(self._nni_experiment_config)
-
-    def get_model_search_update(self):
-        sorted_good_trials = get_experiment_update(self._nni_experiment_config)
-        # re-sort by time
-        metric_values = [s["finalMetricData"] for s in sorted_good_trials]
-        if len(metric_values) == 0:
-            print("no successfully trained models yet")
-        return metric_values[::-1]
-
-    def get_best_model(self):
-        sorted_good_trials = get_experiment_update(self._nni_experiment_config)
-        hyper_parameters = sorted_good_trials[0]["hyperParameters"]
-        model = get_model(hyper_parameters)
-        return model
-
-    def train_best_model(self):
-        model = self.get_best_model()
+        # this only does regressions. choice of model is defined in
+        # the task defininition, though
+        # TODO: pick model type from there from there
```

Comment:
can you please include in this PR (i.e. support classification as well)

Reply:
I've added in classification support along with a very simple (6 row) test dataset in pytest.

I couldn't (in the short time I spent) get this to work with NNI, so behaviour may be different from before.